cmd/agent/app/options/options.go

```diff
@@ -127,11 +115,37 @@ type Options struct {
 // NewOptions builds an default scheduler options.
 func NewOptions() *Options {
 	return &Options{
+		Controllers: []string{"*"},
 		LeaderElection: componentbaseconfig.LeaderElectionConfiguration{
 			LeaderElect:       true,
 			ResourceLock:      resourcelock.LeasesResourceLock,
 			ResourceNamespace: util.NamespaceKarmadaSystem,
+			LeaseDuration:     metav1.Duration{Duration: 15 * time.Second},
+			RenewDeadline:     metav1.Duration{Duration: 10 * time.Second},
+			RetryPeriod:       metav1.Duration{Duration: 2 * time.Second},
 		},
+		BindAddress:                        "0.0.0.0",
+		SecurePort:                         10357,
```

Comment:
Perhaps, we can rank them by ordered alphabets.

Reply:
ok